theme/layout/partials/sidebar.ejs

```diff
@@ -69,31 +69,9 @@
   </div>
 
   <div class="sidebar-content">
-    <div class="topcap">
-      <!--<span class="title-sidebar"><%- config.title %></span>
-      <% if (config.versions && config.versions.length > 1) { %>
-        <span class="select version-sidebar nochrome">
-          <select class="version-select">
-            <% config.versions.forEach(function (v) { %>
-              <option value="v<%- v %>"><%- v %></option>
-            <% }) %>
-          </select>
-        </span>
-      <% } %>
-      -->
-    </div>
 
     <% if (config.apis && config.apis.docsearch) { %>
-    <div class="wrapper-search">
-      <div class="input-area">
-        <div class="input-symbol small round">
-          <input type="text" placeholder="Search <%- config.subtitle %>" id="desktop-search-input" />
-          <span class="icon-search"></span>
-        </div>
-      </div>
-      <div class="wrapper-desktop-search-results"></div>
-    </div>
-
+      <div id="docsearch"></div>
```

Comment:
**nitpick:** removing the `.wrapper-search` removed some vertical spacing too.
I find the new spacing a bit too small.

![screenshot-2022-03-07_18-58-58](https://user-images.githubusercontent.com/75968/157091839-b7b3d1b4-a45c-43ae-ab4e-047186e5c863.png)
![screenshot-2022-03-07_18-57-22](https://user-images.githubusercontent.com/75968/157091841-ee02a83c-1da1-42da-9181-2867011185bd.png)


**suggestion:** add more spacing below the search. In CSS or with
```suggestion
    <div class="wrapper-search">
        <div id="docsearch"></div>
    </div>
```

Reply:
> I noticed a regression from this config, for a person who wants to access to the migration guides. Could you check what is the config behind it?

I found that the new version is not successfully crawling the migration guide as it's complaining that there are too many records 🦘 
![image](https://user-images.githubusercontent.com/39598117/157106175-59e90c34-5493-44a8-8101-320f2fb124af.png)
![image](https://user-images.githubusercontent.com/39598117/157106252-fe58c48d-53fd-4bd6-a216-7d82ede85048.png)

<br />
<br />

> nitpick: removing the .wrapper-search removed some vertical spacing too.
I find the new spacing a bit too small.

Nice catch! I'm going to call you hawkeye 🦅 from now on hehe I'll fix that.

Reply:
I created an additional step when crawling the migration guide to only pull the tagged headers
![image](https://user-images.githubusercontent.com/39598117/157109821-bf5155cc-710d-4579-b677-ceec6ab34d0c.png)

Reply:
Updated to spacing aswell